src/image_function.cpp

```diff
@@ -58,19 +58,22 @@ namespace
         Image_Function::ParameterValidation( image, x, y, width, height );
         Image_Function::VerifyGrayScaleImage( image );
 
-
+	 if ( dilationX == 0u && dilationY == 0u ) 
```

Comment:
@theoniko could you please check indentation of lines. Here we have 9 spaces instead of 8 :)